api/tests/src/api/extracts_v1/test_extract_schema.py

```diff
@@ -0,0 +1,110 @@
+from datetime import date
+
+import pytest
+from marshmallow import ValidationError
+
+from src.api.extracts_v1.extract_schema import (
+    ExtractMetadataListResponseSchema,
+    ExtractMetadataRequestSchema,
+    ExtractMetadataResponseSchema,
+)
+from src.db.models.extract_models import ExtractMetadata
+
+
+@pytest.fixture
+def sample_extract_metadata():
+    return ExtractMetadata(
+        extract_metadata_id=1,
+        extract_type="opportunities_csv",
+        file_name="test_extract.csv",
+        file_path="/test/path/test_extract.csv",
+        file_size_bytes=2048,
+    )
+
+
+def test_request_schema_validation():
+    schema = ExtractMetadataRequestSchema()
+
+    # Test valid data
+    valid_data = {
+        "filters": {
+            "extract_type": "opportunities_csv",
+            "start_date": "2023-10-01",
+            "end_date": "2023-10-07",
+        },
+        "pagination": {
+            "order_by": "created_at",
+            "page_offset": 1,
+            "page_size": 25,
+            "sort_direction": "ascending",
+        },
+    }
+    result = schema.load(valid_data)
+    assert result["filters"]["extract_type"] == "opportunities_csv"
+    assert result["filters"]["start_date"] == date(2023, 10, 1)
+    assert result["filters"]["end_date"] == date(2023, 10, 7)
+
+    # Test invalid extract_type
+    invalid_data = {"extract_type": "invalid_type", "start_date": "2023-10-01"}
+    with pytest.raises(ValidationError):
+        schema.load(invalid_data)
+
+
+def test_response_schema_single(sample_extract_metadata):
+    schema = ExtractMetadataResponseSchema()
+
+    extract_metadata = schema.dump(sample_extract_metadata)
+
+    # The backend will set this as a S3 signed URL
+    extract_metadata["download_path"] = "http://www.example.com"
```

Comment:
You would want to set it before, not after running it through the logic. Like how we do for presigning: https://github.com/HHS/simpler-grants-gov/blob/main/api/src/services/opportunities_v1/get_opportunity.py#L60
```suggestion
    
    sample_extract_metadata.download_path = "http://www.example.com"
    extract_metadata = schema.dump(sample_extract_metadata)


    assert extract_metadata["download_path"] == "http://www.example.com"
```